File: friend-service/src/main/java/vn/iostar/friendservice/dto/response/UserResponse.java
package vn.iostar.friendservice.dto.response;

import lombok.Data;
import vn.iostar.friendservice.constant.Gender;
import vn.iostar.friendservice.constant.RoleName;

import java.util.Date;
@Data
public class UserResponse {
    private String userId;
    private String userName;
    private String address;
    private String phone;
    private Gender gender;
    private Date dayOfBirth;
    private Boolean isActive;
    private RoleName roleName;
    private String email;
    private Boolean status;
    private Boolean isOnline;
    private Long countPost;
    private Long countShare;
    private Long countComment;
}

File: conversation-service/src/utils/DTO/UserResponse.js
class UserResponse {
  constructor({
    userId,
    phone,
    email,
    userName,
    avatar,
    background,
    address,
    dayOfBirth,
    about,
    gender,
    isActive,
    createdAt,
    updatedAt,
    roleName,
    friends,
    postGroup,
    accountActive,
  }) {
    this.userId = userId;
    this.phone = phone;
    this.email = email;
    this.userName = userName;
    this.avatar = avatar;
    this.background = background;
    this.address = address;
    this.dayOfBirth = dayOfBirth;
    this.about = about;
    this.gender = gender;
    this.isActive = isActive;
    this.createdAt = createdAt;
    this.updatedAt = updatedAt;
    this.roleName = roleName;
    this.friends = friends;
    this.postGroup = postGroup;
    this.accountActive = accountActive;
  }
}

module.exports = UserResponse;
